broker/service/api/v10.py

```diff
@@ -57,19 +57,20 @@ def run_submission(data):
 
 
 def stop_submission(submission_id, data):
-    if 'username' not in data or 'password' not in data:
-        API_LOG.log("Missing parameters in request")
-        raise ex.BadRequestException()
-    
-    username = data['username']
-    password = data['password']
+    if data['enable_auth']:
```

Comment:
Should we check for 'enable_auth' before accessing it on data?

Reply:
Yes, this is a temporary solution. Because the authentication resources isn't correctly implemented. The body of submission need have a property "enable_auth" : False to ignore the authentication.